modules/terms/partials/acl.adoc

```diff
@@ -0,0 +1,4 @@
+=== access-control list (ACL) 
+:term-name: ACL
+:hover-text: A security feature used to define and enforce granular permissions to resources, ensuring only authorized users or applications can perform specific operations. ACLs act on principals. 
+:link: https://docs.redpanda.com/current/manage/security/authorization/
```

Comment:
@JakeSCahill also not sure if this is how to add internal links.

Reply:
I want these as see also type links in the glossary page (not a direct link like in the [existing cert-manger term](https://docs.redpanda.com/current/manage/kubernetes/security/tls/use-cert-manager/). Should I remove internal links from the :link: attribute and just put them as See also text under the hover text?

Reply:
Yes 👍 The default behavior links to the glossary page. The purpose of the `link` attribute is to override that behavior to link to an external site for the definition.